src/@chakra-ui/gatsby-plugin/components/Table.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentStyleConfig } from "@chakra-ui/react"
+
+export const Table: ComponentStyleConfig = {
```

Comment:
Should we keep the old margins here @nloureiro?


Old:

<img width="1043" alt="Screenshot 2022-10-03 at 10 55 34" src="https://user-images.githubusercontent.com/62268199/193550135-754c9fe5-2e26-4321-b3e6-0bcc0027d7ca.png">

This PR:

<img width="1105" alt="Screenshot 2022-10-03 at 10 55 48" src="https://user-images.githubusercontent.com/62268199/193550140-800f66eb-181b-4b42-8042-3117722fcf0c.png">